chat-widget/src/common/Constants.ts

```diff
@@ -65,7 +65,9 @@ export class Constants {
         "application/vnd.microsoft.card.thumbnail",
         "application/vnd.microsoft.card.signin",
         "application/vnd.microsoft.card.oauth",
+        "application/vnd.microsoft.card.video"
     ];
+    public static readonly adaptiveCardContentTypePrefix = "application/vnd.microsoft.card";
```

Comment:
We are checking "application/vnd.microsoft.card" as unsupported prefix format, but in the above list all of them has same prefix saying "supportedAdaptiveCardContentTypes".